src/enrichment.cc

```diff
@@ -292,6 +298,9 @@ void Enrichment::GetMatlTrades(
                              " is being asked to provide more than" +
                              " its SWU capacity.");
   }
+
+  RecordTimeSeries<double>("supply"+tails_commod, this, tails_supply_qty);
```

Comment:
same comment as with demand above. What is the purpose of the "supply" string here? Is the string prepended with this to create a meaningful data feature in RecordTimeSeries, or would tails_commod be just as good?

Reply:
so the table names for RecordTimeSeries are prepended with either `supply` or `demand` to let `d3ploy` know that this table is for either supply or demand for that commodity. For example, a reactor would record `timeseriesdemanduox` while a source records `timeseriessupplyuox`.